services/src/main/java/org/exoplatform/addons/gamification/utils/Utils.java

```diff
@@ -201,18 +208,25 @@ public static UserInfo getUserById(Long id, Long challengeId) {
   }
 
   public static UserInfo createUser(Identity identity, Space space, List<Long> managersId) {
-    SpaceService spaceService = CommonsUtils.getService(SpaceService.class);
     UserInfo userInfo = new UserInfo();
     userInfo.setAvatarUrl(identity.getProfile().getAvatarUrl());
     userInfo.setFullName(identity.getProfile().getFullName());
     userInfo.setRemoteId(identity.getRemoteId());
     userInfo.setId(identity.getId());
+    String userId = getCurrentUser();
```

Comment:
Are you sure about this ? this will retrurn current user right ?

Reply:
Right ! fixed!